### PR TITLE
Streams egress best practices content

### PR DESCRIPTION
**Pull request by GiselleLeeOSI**
https://dev.azure.com/osieng/engineering/_sprints/taskboard/guild.Content%20Finn/engineering/2021/Q1/2021.03.03?workitem=192316

### CODE REVIEW

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md [hunk 1]
```diff
@@ -42,16 +42,12 @@ This maximum limit applies to [Get Values](xref:sdsReadingDataApi#get-values), [
...
 
-If the read data API needs to hold events larger than 2GB in memory size, a ``500 - Internal server error`` will be returned. 
-
- 
-
 ### Increase the Request-Timeout in header 
 
 Increase the Request-Timeout in the header to 5 minutes for large range calls that are requesting 250,000 events in a read call. 
 The gateway will send ``408 - Operation timed out error`` if the request needs more than 30 seconds. 
 It is possible that these large range requests will take up to 5 minutes.
```
Comment (hdittakavi) [inline Content_Portal/add-organize-data/store-data/streams-best-practices.md:49]: We can remove this line. It has been removed in the doc.

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md [hunk 2]
```diff
@@ -42,16 +42,12 @@ This maximum limit applies to [Get Values](xref:sdsReadingDataApi#get-values), [
...
-The large range of values when held in memory are between 1 GB and 2GB hence need enough time to read and return the data.  
+The large range of values that are held in memory are between 1 GB and 2GB so the system needs enough time to read and return the data.  
 
 If multiple calls return ``408 - Operation timed out error`` even after increasing the timeout limit to 300,000 milliseconds (= 300 seconds = 5 minutes), do one of the following:
```
Comment (hdittakavi) [inline Content_Portal/add-organize-data/store-data/streams-best-practices.md:52]: Let's just keep this to one kind of metric which is 5 minutes as it is in the above line. Updated in the doc too now.

Comment (hdittakavi) [reply Content_Portal/add-organize-data/store-data/streams-best-practices.md:52]: Line 55: Replace "Hold off on sending these requests and contact OSIsoft technical support" with "Retry with an exponential back-off policy. "

File: Content_Portal/add-organize-data/store-data/streams-best-practices.md [hunk 3]
```diff
@@ -17,3 +17,59 @@ OSIsoft recommends the following are best practices when creating streams:
...
+
+} 
+```
+
+### Increase the Request-Timeout in header 
+
+Increase the Request-Timeout in the header to 5 minutes for large range calls that are requesting 250,000 events in a read call. 
+The gateway will send ``408 - Operation timed out error`` if the request needs more than 30 seconds. 
+
+The large range of values that are held in memory are between 1 GB and 2GB so the system needs enough time to read and return the data.
```
Comment (hdittakavi) [inline Content_Portal/add-organize-data/store-data/streams-best-practices.md:50]: can be

**Comment by joyceseng** [comment]
Hi there, Ji Won! I should be able to look at this tomorrow (Tues). Thanks!

**Comment by GiselleLeeOSI** [comment]
Closing PR and moving the content to developer documentation

**Pull request closed without merging**